third_party/llvm-project/include/llvm/DebugInfo/DWARF/DWARFDebugLine.h

```diff
@@ -266,7 +266,10 @@ class DWARFDebugLine {
     bool getFileNameByIndex(uint64_t FileIndex, StringRef CompDir,
                             DILineInfoSpecifier::FileLineInfoKind Kind,
                             std::string &Result) const {
-      return Prologue.getFileNameByIndex(FileIndex, CompDir, Kind, Result);
+      return Prologue.getFileNameByIndex(FileIndex, CompDir, Kind, Result,
+        // FIXME: this fixes paths being incorrect when generated on
+        // Windows, but needs to be upstreamed how?
+        llvm::sys::path::Style::posix);
```

Comment:
Do we need this? I thought the test was disabled earlier.

We considered doing this in the past, but were worried that it may change observable behavior for emscripten users with DWARF. If we change this we may need to make a corresponding change on the emscripten side, I'm not sure, but hopefully we can avoid both.

Reply:
This would allow us to enable those tests on Windows. I will remove it for now, since at least we now know exactly where that `\` gets introduced..